.vib/discourse/cypress/cypress/integration/discourse_spec.js

```diff
@@ -0,0 +1,64 @@
+/// <reference types="cypress" />
+import { random } from './utils';
+
+it('allows to log in and out', () => {
+  cy.login();
+  cy.contains('a', 'Unread');
+  cy.get('#current-user').click();
+  cy.get('[title="Preferences"]').click();
+  cy.contains('Log Out').click();
+  cy.contains('button', 'Log In');
+});
+
+it('allows to sign in', () => {
+  cy.contains('button', 'Sign Up').click();
```

Comment:
This test is not independent.
It works because you execute it after the first one. Since there is no login here, the executed test will fail in isolation. 
Just add a login before and it will be OK.